speechmatics_flow/models.py

```diff
@@ -179,7 +191,10 @@ class ServerMessageType(str, Enum):
     """Indicates a generic warning message."""
 
     Error = "Error"
-    """Indicates n generic error message."""
+    """Indicates a generic error message."""
+
+    Debug = "Debug"
+    """Indicates a debug message based on the flag set in the StartConversation message"""
```

Comment:
```suggestion
    """Indicates a debug message"""
```
I know it doesn't sound very descriptive, but I would l just say this. We have undocumented `Debug` messages already going over the websocket that cannot be configured from `StartConversation`